memflow/Cargo.toml

```diff
@@ -43,13 +44,25 @@ smallvec = { version = "1.13", default-features = false }
 libloading = { version = "0.8", optional = true }
 dirs = { version = "5.0", optional = true }
 # enable unsafe_alignment for now, this is UB, but the idea is to fix pelite in upstream
-pelite = { version = "=0.9.0", optional = true, default-features = false, features = ["unsafe_alignment"] }
+pelite = { version = "=0.9.0", optional = true, default-features = false, features = [
```

Comment:
Can you remove the auto formatting for the Cargo.toml file please? There's a lot of formatting changes here